frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -61,8 +90,8 @@ export function PersonModal({ visible, view, filters, onSaveCohort }: PersonModa
         [filters, people, isInitialLoad]
     )
 
-    const isDownloadCsvAvailable = view === ViewType.TRENDS
```

Comment:
Why remove CSV download on Postgres Trends?

Reply:
just removed the variable but the condition is wrapped in the one below. So wherever you can save as cohort you can download CSV